tests/utils/shippable/timing.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3.7
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
```

Comment:
If it's supposed to run under Python 3.7, it doesn't make sense to have this boilerplate:
```suggestion
#!/usr/bin/env python3.7
```